### DISCUSSION TITLE
Ultralytics Live Session 5: Tracking Your Way To Computer Vision Success

### DISCUSSION THREAD

**Discussion by taliabender** [ultralytics/ultralytics#1120] [Show and tell]
[Kristen Kehrer](https://www.linkedin.com/in/ACoAAAZ4apwBPjZJ-x-X9_2mIA6aYXQo4b_LEmM) and [Glenn Jocher](https://www.linkedin.com/in/ACoAAASa684BOUpWeSgSJbXKXU7MaWJqS4RCFes) have something in common: their unconventional journeys into the world of vision AI.

The pace of change in AI can be overwhelming: with numerous frameworks, tools, and libraries to choose from, it can be difficult to know where to start.

In our next Live Session, we'll be breaking down how Kristen and Glenn got their start with AI, tips & strategies for YOLOv5 and YOLOv8 model training, and the ins and outs of experiment tracking with [Comet](https://www.linkedin.com/company/comet-ml/). 🔥

Register now and mark your calendars!

📆 March 12th
⏰ 12:00 EST/18:00 CET
🔗 https://lnkd.in/dUb4AyPr

![ULS Ep5 Comet TW:LI](https://user-images.githubusercontent.com/85292283/220964191-b8c63160-16ab-4216-83e5-31577413991f.png)

**Comment by glenn-jocher** [comment]
@taliabender hello there! 👋

Thank you for sharing the excitement about the upcoming Live Session. It's always a pleasure to discuss the dynamic field of vision AI and share insights with the community.

In this session, we'll indeed explore the journeys that led Kristen and myself into AI, and we'll offer tips and strategies specifically for YOLOv8 model training. While YOLOv5 has its own set of features and community, our focus will be on the advancements and capabilities of YOLOv8.

For those interested in the nitty-gritty of experiment tracking, we'll also delve into how tools like Comet can streamline the process, making it easier to manage and optimize your models.

Remember, if you're looking for detailed documentation on YOLOv8, including how to train, validate, and deploy your models, you can always visit our comprehensive docs at https://docs.ultralytics.com.

Looking forward to seeing you on March 12th! Your participation and questions will surely enrich the session. 😊
